db/fixtures/miq_user_roles.yml

```diff
@@ -75,7 +75,7 @@
   - physical_storage
   - physical_infra_topology
   - policy_import_export
-  - provider_foreman_explorer
+  - configuration_manager_explorer
```

Comment:
Wait, is this still existing? Don't you want to replace it the new 3 features instead?

Reply:
good catch, will take care of that

Reply:
@skateman done, replaced old features with new ones.